### DISCUSSION TITLE
Stop emitter from producing when limit is reached instead of destroying

### DISCUSSION THREAD

**Discussion by JusticeMatthew** [matteobruni/tsparticles#4932] [Q&A] [answered]
Hello,

I've scoured the web trying to find out how to do this but I can't seem to get it working.

I have an `emitter`, I want it to emit 10 particles and then stop.

Using the `limit` property just causes it to destroy a particle to create a new one keeping the total number of 10, but I just want the 10 original particles to stay alive.

Thanks!

**Comment by matteobruni** [comment, marked as answer]
Emitters rate and life options can be adjusted for this behavior. Set a life count of 1, life duration 10 times the rate delay, with a rate of 1. Divide 10 by the rate if different value

**Reply by JusticeMatthew** [reply]
Thanks very much!

I had played with the life property but clearly lacked the understanding haha
